rust/crd/src/lib.rs

```diff
@@ -295,6 +322,8 @@ pub struct DriverConfig {
     pub memory: Option<String>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub volume_mounts: Option<Vec<VolumeMount>>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
Isn't this already covered by the existing volume mechanism?

Reply:
I think the difference here is that we are not delegating a mechanism known to kubernetes, but rather defining a pod template that will be used by spark internally to create the driver and executor pods.

Reply:
Not sure I follow.. looks like they're both serialized to regular volume mounts in the code?

Reply:
Ah, I misunderstood. Yes, I can use a VolumeMount rather than a new struct. I'll change that.